### ISSUE TITLE
Tensorboard not displaying Graphs Anymore

### ISSUE THREAD

**Issue by cmbowyer13** [tensorflow/tensorboard#949]
/Users/.../anaconda/lib/python3.6/importlib/_bootstrap.py:205: RuntimeWarning: compiletime version 3.5 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.6

**Comment by chihuahua** [comment]
Could you please elaborate on which graphs are not displaying? And perhaps include a screenshot? Thank you!

**Comment by peteboothroyd** [comment]
I'm also having the same problem with Tensorboard failing to display graphs, specifically in chrome. However if I try to view the graph in Safari this works fine. I'm on macOS high sierra, and using Tensorflow r1.5 built from source.

**Comment by nfelt** [comment]
Thanks for the report.  Could you provide your Chrome version number (`chrome://version`) and also check the javascript console in Chrome to see if it's reporting any errors, and if so include those as well?

Also, are you seeing this for any graph at all or just for certain graphs?

**Comment by peteboothroyd** [comment]
I've just checked again and it began working fine for Chrome version 64.0.3282.186. However it then stopped working. There was an uncaught type error when clicking around the graph when it was working. 
``` javascript
Uncaught TypeError: pathNode.getPathSegAtLength is not a function
    at adjustPathPointsForMarker (tf-tensorboard.html:10555)
    at SVGPathElement.getEdgePathInterpolator (tf-tensorboard.html:10648)
    at SVGPathElement.attrTween (d3.js:8772)
    at d3_Map.<anonymous> (d3.js:8945)
    at d3_Map.forEach (d3.js:341)
    at start (d3.js:8944)
    at Object.schedule (d3.js:8912)
    at d3_timer_mark (d3.js:2165)
    at d3_timer_step (d3.js:2146)
    at web-animations-next-lite.min.js:16
adjustPathPointsForMarker @ tf-tensorboard.html:10555
getEdgePathInterpolator @ tf-tensorboard.html:10648
attrTween @ d3.js:8772
(anonymous) @ d3.js:8945
forEach @ d3.js:341
start @ d3.js:8944
schedule @ d3.js:8912
d3_timer_mark @ d3.js:2165
d3_timer_step @ d3.js:2146
(anonymous) @ web-animations-next-lite.min.js:16
requestAnimationFrame (async)
window.requestAnimationFrame @ web-animations-next-lite.min.js:16
d3_timer @ d3.js:2141
d3_transitionNode @ d3.js:8965
d3_selectionPrototype.transition @ d3.js:8637
update @ tf-tensorboard.html:10291
(anonymous) @ tf-tensorboard.html:10195
(anonymous) @ d3.js:961
d3_selection_each @ d3.js:967
d3_selectionPrototype.each @ d3.js:960
buildGroup @ tf-tensorboard.html:10193
(anonymous) @ tf-tensorboard.html:10805
(anonymous) @ d3.js:961
d3_selection_each @ d3.js:967
d3_selectionPrototype.each @ d3.js:960
buildGroup @ tf-tensorboard.html:10796
buildGroup @ tf-tensorboard.html:11992
(anonymous) @ tf-tensorboard.html:13725
time @ tf-tensorboard.html:12430
_build @ tf-tensorboard.html:13724
setNodeExpanded @ tf-tensorboard.html:13697
(anonymous) @ tf-tensorboard.html:14207
(anonymous) @ polymer.html:1347
setTimeout (async)
run @ polymer-mini.html:2014
async @ polymer.html:1346
_nodeToggleExpand @ tf-tensorboard.html:14206
handler @ polymer.html:561
fire @ polymer.html:1327
(anonymous) @ tf-tensorboard.html:10895
(anonymous) @ d3.js:1119
```

**Comment by nfelt** [comment]
Hmm, the `getPathSegAtLength()` error sounds like #425 which should be fixed in TensorBoard 1.5.0+.  Can you confirm that your TensorBoard version is at least that high?  You might also try installing TensorBoard 1.6.0 (`pip uninstall tensorflow-tensorboard && pip install tensorboard`).

And just to confirm, do you see this even with a very simple graph?  For example, this will create a graph with a single "foo" node:

```python
import tensorflow as tf
tf.Variable(42, name='foo')
w = tf.summary.FileWriter('/tmp/testdir')
w.add_graph(tf.get_default_graph())
w.flush()
w.close()
```

Then you can run `tensorboard --logdir /tmp/testdir` - does the graph still show up as empty?

**Comment by peteboothroyd** [comment]
Ah I think that was it, I have updated to Tensorboard 1.6 and it looks ok now. Thanks.

**Comment by nfelt** [comment]
Great, glad to hear it!

**Comment by kikinewton** [comment]
hello, am a tensorflow noob and i have tried several times to display a graph with tensorboard but am having issues. It always displays this image though i have followed the documentation on the official page.
![screenshot 177](https://user-images.githubusercontent.com/15596612/36902564-d9426860-1e22-11e8-8701-6bcbfd8cd0db.png)

**Comment by nfelt** [comment]
@kikinewton What is the command line you're using to run tensorboard, and what is the specific directory in which you're running that code with the FileWriter - i.e. where do your event files end up?

**Comment by kikinewton** [comment]
am running the anaconda prompt and my events are being saved in a local directory on my user folder. The folder is called my_graph

**Comment by nfelt** [comment]
Can you provide the exact tensorboard command you're running?  You might need to include a drive name in the `--logdir` flag argument, e.g. try `--logdir C:\Users\Kiki\my_graph`.  Can you also try running tensorboard with the `--inspect` flag and let us know what the result is?

**Comment by winnechan** [comment]
Hi, my graph appears a second and then disappers in the browser. After that, I want to export the PNG image and find that the image is 0 KB and the title says "graph_large_attrs_key=_too_large_attrs&limit_attr_size=1024&run=".
How to solve such a problem? Thanks!

**Comment by rakababa** [comment]
I get a blank white screen when i run tensorboard (v 1.13.1). Also running --inspect gives me the following:

![image](https://user-images.githubusercontent.com/23608242/55779943-a64b7e80-5a5b-11e9-8d8b-f662d672b5b4.png)

Is there something wrong in my events file??

**Comment by amanp592** [comment]
@rakababa  can you provide the code that is producing this issue?? it will be easier to help then.

**Comment by dibakarray** [comment]
My graph appeared once I switched to Chrome.

**Comment by mehedi-tut** [comment]
> Hmm, the `getPathSegAtLength()` error sounds like #425 which should be fixed in TensorBoard 1.5.0+. Can you confirm that your TensorBoard version is at least that high? You might also try installing TensorBoard 1.6.0 (`pip uninstall tensorflow-tensorboard && pip install tensorboard`).
> 
> And just to confirm, do you see this even with a very simple graph? For example, this will create a graph with a single "foo" node:
> 
> ```python
> import tensorflow as tf
> tf.Variable(42, name='foo')
> w = tf.summary.FileWriter('/tmp/testdir')
> w.add_graph(tf.get_default_graph())
> w.flush()
> w.close()
> ```
> 
> Then you can run `tensorboard --logdir /tmp/testdir` - does the graph still show up as empty?

In my case, even this script does not run tensorboard in chrome. Could you please tell me the reason.

**Comment by stephanwlee** [comment]
@mehedi-tut this issue is quite dated and context is quite hard to follow. Could you please file another bug with better description of your issue? Also, please attach requested information in the new issue template.

**Comment by mehedi-tut** [comment]
> @mehedi-tut this issue is quite dated and context is quite hard to follow. Could you please file another bug with better description of your issue? Also, please attach requested information in the new issue template.

Yes, I will. Thanks.

**Comment by xukkTodd** [comment]
In my case, the graph cannot be displayed because it is too long. You need to zoom in to see the graph.
